src/packages/@ncigdc/tableModels/utils.js

```diff
@@ -109,9 +109,9 @@ export const createSelectColumn: TCreateSelectColumn = ({
     downloadable: false,
     hidden: false,
     th: ({ nodes, selectedIds, setSelectedIds }) => {
-      const ids = nodes.map(node => node[idField]);
+      //TODO: make it easier to understand the structure of nodes obj
```

Comment:
maybe instead:

`// NOTE: "nodes" is really "edges" in the graphql schema`

Reply:
Or reaname it edges?